help/status-and-availability.md

```diff
@@ -48,12 +48,38 @@ You can set a status emoji, status message, or both.
 ### View a status
 
 Status emoji are shown next to a user's name in the sidebars, message feed,
-and compose box.
+and compose box. Status emoji and status messages are also shown on
+[user cards](/help/user-cards).
 
-You can view status messages by hovering over your or anyone else's name in the
-left or right sidebar, or by clicking the user's name or avatar in the main
-message feed to open their **user card**. If someone hasn't set a message as
-part of their status, then no status message will appear.
+{start_tabs}
+
+{tab|desktop-web}
+
+1. Hover over a user's name in the right sidebar, or in the **Direct messages**
+   section in the left sidebar to view their status message. You can also click
+   on a user's profile picture or name on a message they sent to open their
+   **user card**.
```

Comment:
How about something like this?

> to view their status in their **user card**.

Otherwise, it's not super clear why you'd want to open the user card.

Reply:
Do you think this alternative instruction would be better as a tip? I feel like this is a really complicated step as it's currently written.

Reply:
Yeah, I was thinking about that too. So we would end up with two tips in each tab? Hm, I think that would be reasonable in this case since it would improve readability.

Reply:
Yeah, or we could drop the tip about empty status messages, and make the instruction something like "... to view their status message if they have one set."

Reply:
@alya I updated the PR/screenshots.